src/validation/Validator.ts

```diff
@@ -332,6 +334,17 @@ export class Validator {
         return value instanceof Boolean || typeof value === "boolean";
     }
 
+    isLatLong(value: any): boolean {
```

Comment:
Why did you copy/paste solution from validator.js? We have instance of validator.js already created. Use `this.validatorJs.isLatLong(...)`

Reply:
i was thinking of doing that but wanted the patch to look original. I can fix that if you like

Reply:
It is not a patch. We use validatorjs internally all the time.

Reply:
okay then shall i update the code or let it be

Reply:
Update. Use validator.js function. Check e.g. `isDivisibleBy` where we use validator.js function too.

Reply:
ok on it

Reply:
@vlapo fixed it